index.html

```diff
@@ -73,8 +73,246 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
             </section>
         </div>
 
+    </div>
+    <div class="cards grid-container">
+        <section class="first-card card">
+            <img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+            <section>
+                <h1 class="card-header font-style">
+                    Tonic
+                </h1>
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).